### PR TITLE
chore(deps): bump trivy-iac and rm trivy-policies dep

### PR DESCRIPTION
**Pull request by simar7**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -143,3 +143,7 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/aquasecurity/trivy-policies => /Users/simarpreetsingh/repos/trivy-policies
+
+replace github.com/aquasecurity/trivy-iac => /Users/simarpreetsingh/repos/trivy-iac
```
Comment (simar7) [inline go.mod:149]: remove when new trivy-policies,iac are released

File: pkg/scanner/scanner.go [hunk 2]
```diff
@@ -206,13 +206,6 @@ func (s *Scanner) Scan(ctx context.Context, cloudState *state.State) (results sc
 	return append(results, regoResults...), nil
 }
 
-func (s *Scanner) getRules() []types.RegisteredRule {
-	if len(s.frameworks) > 0 { // Only for maintaining backwards compat
-		return rules.GetRegistered(s.frameworks...)
-	}
-	return rules.GetSpecRules(s.spec)
-}
-
```
Comment (nikpivkin) [inline pkg/scanner/scanner.go:215]: I think we can revert this and import RegisteredRule from trivy-iac.

**Review by nikpivkin — CHANGES_REQUESTED** [review]

File: pkg/scanner/scanner.go (hunk 2 above)
Comment (simar7) [reply pkg/scanner/scanner.go]: did so https://github.com/aquasecurity/trivy-aws/pull/35/commits/b89962ae3b5231d3c77b7a71e3bcd18d6f24c3a0

File: pkg/scanner/scanner_test.go [hunk 3]
```diff
@@ -17,47 +18,79 @@ import (
...
 	"github.com/stretchr/testify/require"
 )
 
+type testStruct struct {
+	name      string
+	scanner   *Scanner
+	fwApplied framework.Framework
+}
+
 func TestScanner_GetRegisteredRules(t *testing.T) {
```
Comment (nikpivkin) [inline pkg/scanner/scanner_test.go:32]: I think the tests can also be reverted

**Pull request merged**